backend/src/main/java/com/redhat/cloud/notifications/db/BehaviorGroupResources.java

```diff
@@ -55,30 +70,83 @@
         });
     }
 
-    // TODO Should this be forbidden for default behavior groups?
     public Uni<Boolean> update(String accountId, BehaviorGroup behaviorGroup) {
+        return this.update(accountId, behaviorGroup, false);
+    }
+
+    public Uni<Boolean> updateDefault(BehaviorGroup behaviorGroup) {
+        return this.update(null, behaviorGroup, true);
+    }
```

Comment:
This is never used. Should it be removed?

Reply:
Being used now!